### ISSUE TITLE
Setup spotify in the simulator

### ISSUE THREAD

**Issue by ConnorFoody** [me-next/menext-app#49]
Might be faster deploys, needed for #41 unless we want to mock stuff

**Comment by daniel-centore** [comment]
AFAIK, we cannot make Spotify work in the simulator.

Alternatives:
1) Mock (which I think is perfectly acceptable).
2) If Spotify is not installed, open a browser instead and get the session from Spotify website. This is officially supported but I might need to do some restructuring to make it work.
3) By default play the song preview, which can be done without signing in. However, it requires completely different code and I don't really see a good reason to put in the effort for this.

**Comment by daniel-centore** [comment]
For now, I will structure the API to just do nothing if it cannot play a song, rather than err'ing (i.e. alternative 1). This unblocks #41. We can consider the more complex solution later.

**Comment by ConnorFoody** [comment]
sounds like a plan

**Comment by daniel-centore** [comment]
In fact, I've actually had the code working already for alternative 2 (b/c I first made a near-copy of the example project) but it requires that we take over as the UI controller and so I'd need to figure out how to make it play nice with the rest of the project.

**Comment by daniel-centore** [comment]
Closing in favour of #63. We will continue to develop under the assumption of mocking until #63 is resolved.
